ui/src/main.tsx

```diff
@@ -1,10 +1,17 @@
-import React from 'react';
-import ReactDOM from 'react-dom/client';
-import { App } from './App.tsx';
-import './index.css';
+import React from "react";
```

Comment:
organize imports pls
so libs on top, styles are the last

Reply:
Imports organize